gui/wxpython/nviz/tools.py

```diff
@@ -734,24 +734,12 @@ def _createDataPage(self):
         self.mainPanelData = SP.ScrolledPanel(parent=self)
         self.mainPanelData.SetupScrolling(scroll_x=False)
         self.mainPanelData.AlwaysShowScrollbars(hflag=False)
-        try:  # wxpython <= 2.8.10
-            self.foldpanelData = fpb.FoldPanelBar(
-                parent=self.mainPanelData,
-                id=wx.ID_ANY,
-                style=fpb.FPB_DEFAULT_STYLE,
-                extraStyle=fpb.FPB_SINGLE_FOLD,
-            )
-        except:
-            try:  # wxpython >= 2.8.11
-                self.foldpanelData = fpb.FoldPanelBar(
-                    parent=self.mainPanelData,
-                    id=wx.ID_ANY,
-                    agwStyle=fpb.FPB_SINGLE_FOLD,
-                )
-            except:  # to be sure
-                self.foldpanelData = fpb.FoldPanelBar(
-                    parent=self.mainPanelData, id=wx.ID_ANY, style=fpb.FPB_SINGLE_FOLD
-                )
+        self.foldpanelData = fpb.FoldPanelBar(
+            parent=self.mainPanelData,
+            id=wx.ID_ANY,
+            style=fpb.FPB_SINGLE_FOLD,
```

Comment:
This should match the call with comment "wxpython >= 2.8.11", not a combination of all the different calls...
```suggestion
```